include/circt/Dialect/OM/OMOps.td

```diff
@@ -39,6 +39,18 @@ def ClassOp : OMOp<"class",
     SizedRegion<1>:$body
   );
 
+  let builders = [
+    OpBuilder<(ins "::mlir::Twine":$name,
+                    "::mlir::ArrayRef<::mlir::Attribute>":$params), [{
```

Comment:
I think this can be more specific right? This is for the `formalParamNames` attribute? If that's the case, can its type be `ArrayRef<StringAttr>`? And could we name the `$params` to `$formalParamNames`? We deal with formal and actual parameters, so I think it's important to be religious about the types and names we give to our code to avoid confusion.

Reply:
Updated the name and the type.